index.js

```diff
@@ -0,0 +1,219 @@
+  tripDetail.empty();
+
+  axios.get(URL + id)
+    .then((response) => {
+      let data = response.data;
+      let name = $(`<h4><strong>Name:</strong> ${data.name}</h4>`).addClass(`${id}`);
+      let about = $(`<span><strong>Description:</strong> ${data.about.slice(0, 150)}</span>`).addClass("teaser");
+      let aboutFullText = $(`<span>${data.about.slice(150)}</span>`).addClass("moreinfo hidden").attr('id', 'info1');
+      let more = $(`<span>...Read more</span>`).addClass("more").attr('target', 1);
+      let continent = $(`<p><strong>Continent:</strong> ${data.continent}</p>`);
+      let category = $(`<p><strong>Category:</strong> ${data.category}</p>`);
+      let weeks = $(`<p><strong>Weeks:</strong> ${data.weeks}</p>`);
+      let cost = $(`<p><strong>Cost:</strong> $${data.cost}</p>`);
+
+      tripDetail.append(name, about, aboutFullText, more, continent, category, weeks, cost);
+
+      reportStatus(`Successfully loaded details for ${response.data.name} trip`)
+    })
+    .catch((error) => {
+      console.log(error);
+      reportStatus(`Error: ${error.message}`);
+    });
+}
+
+// Form helpers
+const FORM_FIELDS = ['name', 'email'];
+const inputField = name => $(`#trip-form input[name="${name}"]`);
+
+const readFormData = () => {
+  const getInput = name => {
+    const input = inputField(name).val();
+    return input ? input : undefined;
+  };
+
+  const formData = {};
+  FORM_FIELDS.forEach((field) => {
+    formData[field] = getInput(field);
+  });
+
+  return formData;
+};
+
+const clearForm = () => {
+  FORM_FIELDS.forEach((field) => {
+    inputField(field).val('');
+  });
+}
+
+// Reserve spot on trip
+const reserveTrip = (event) => {
+  event.preventDefault();
+  // let id = Number($('#trip-detail-section h4')[0].classList[0]);
+  let id = $('.trip-detail-section h4').attr("class");
+
+  const tripData = readFormData();
+
+  reportStatus('Sending trip reservation data...');
+
+  axios.post((`${URL}${id}/reservations`), tripData)
+    .then((response) => {
+      clearForm();
+      reportStatus(`Successfully created trip reservation with ID ${response.data.trip_id}!`);
+    })
+    .catch((error) => {
+      console.log(error.response);
+      if (error.response.data && error.response.data.errors) {
+        reportError(
+          `Encountered an error: ${error.message}`,
+          error.response.data.errors
+        );
+      } else {
+        reportStatus(`Encountered an error: ${error.message}`);
+      }
+    });
+}
+
+$(document).ready(() => {
+  $('#load').click(loadTrips);
+  $('#asia').click(asiaTrips);
+  $('#africa').click(africaTrips);
+  $('#antartica').click(antarticaTrips);
+  $('#australasia').click(australasiaTrips);
+  $('#europe').click(europeTrips);
+  $('#north-amer').click(nAmericaTrips);
+  $('#south-amer').click(sAmericaTrips);
+
+  $('#trip-list').on('click', 'li', function(event) {
+    let id = Number(event.target.id);
+    $("section").removeClass("hidden");
```

Comment:
this ends up working, but it might be more specific to use a more specific selector than `'section'` since you have more than one section in your HTML